css/main.css

```diff
@@ -564,11 +414,12 @@ article p {
     color:#595959;
     float:left;
     padding:0;
-    margin:30px 0 0 -180px;
-    width:560px;
+    width:90%;
```

Comment:
Remember that even with box-sizing: border-box; this width will be in addition to the side margins. You'll want to reduce this width to avoid clipping / crowding of the author section at lower screen widths.